### PR TITLE
Move Composite files to easily-found folder

### PR DESCRIPTION
**Pull request by dougbu**
- see discussion starting at <https://github.com/dotnet/aspnetcore/pull/41855#issuecomment-1159030492>
- short answer: wildcards do not work in artifact paths

### CODE REVIEW

File: src/Framework/App.Runtime/src/Microsoft.AspNetCore.App.Runtime.csproj [hunk 1]
```diff
@@ -466,7 +466,7 @@ This package is an internal implementation of the .NET Core SDK and is not meant
     DependsOnTargets="_ExpandRuntimePackageRoot">
 
     <PropertyGroup>
-      <CompositeTargetDir>$(TargetDir)Composite\</CompositeTargetDir>
+      <CompositeTargetDir>$([MSBuild]::NormalizeDirectory('$(ArtifactsObjDir)', 'Composite', '$(Configuration)')</CompositeTargetDir>
```
Comment (trylek) [inline src/Framework/App.Runtime/src/Microsoft.AspNetCore.App.Runtime.csproj:469]: There seems to be a missing second closing parenthesis corresponding to the one after the initial dollar sign, I believe that's why <code>NormalizeDirectory</code> doesn't get expanded and creation of the path fails the build.

**Review by wtgodbe — APPROVED** [review]
LGTM other than Tomas' comment

**Review by trylek — APPROVED** [review]
LGTM, thank you!

**Comment by ivdiazsa** [comment]
Thank you very much for fixing this issue @dougbu! Aside from Tomas' comments (which I see have been addressed), this looks good to me.

**Review by ivdiazsa — APPROVED** [review]

**Comment by dougbu** [comment]
I'll merge this once I confirm the artifact is created correctly. Shouldn't have used a fork ☹️

**Comment by dougbu** [comment]
Files were uploaded in my internal build 😃

**Pull request merged**